charts/consul/values.yaml

```diff
@@ -1859,6 +1859,86 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni pluging for Consul Service mesh services
+  # Using this feature requires Consul 1.13.0-beta+
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+
+    # Namespace to install cni plugin.
+    # @type: string
+    namespace: kube-system
+ 
+    # If multus plugin is enabled or not. If true, the cni installer will generate a consul cni configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multis. cniBinDir and cniNetDir must be changed to work with multus.
+    # Please refer to your cloud platform for more informaton on using multus.
+    # @type: boolean
+    multus: false
+
+    # Log level for the installer and plugin. Overrides global.logLevel
+    # @type: string
+    logLevel: null 
+    
+    # Location on the kubernetes node where the CNI plugin is installed. Shoud be the absolute path and start with a '/'
+    # @type: string
+    cniBinDir: "/opt/cni/bin"
+
+    # Location on the kubernetes node of all CNI configuration. Should be the absolute path and start with a '/'
+    # @type: string
+    cniNetDir: "/etc/cni/net.d"
+
+    # The daemonset should in privileged mode on all kubernetes platforms as it needs to interact with the
+    # kubernetes host file system
+    # @type: boolean
+    privileged: true
+  
+    # The resource settings for cni daemonset.
+    # @recurse: false
+    # @type: map
+    resources:
+      requests:
+        memory: "50Mi"
+        cpu: "50m"
+      limits:
+        memory: "50Mi"
+        cpu: "50m"
+
+    # Resource quotas for running the daemonset as system critical pods
+    resourceQuota:
+      pods: 5000
+
+    # The security context for the cni installer daemonset. This should be a YAML map corresponding to a
+    # Kubernetes [SecurityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) object.
+    # By default, servers will run as non-root, with user ID `100` and group ID `1000`,
+    # which correspond to the consul user and group created by the Consul docker image.
+    # Note: if running on OpenShift, this setting is ignored because the user and group are set automatically
+    # by the OpenShift platform.
+    # @type: map
+    # @recurse: false
+    securityContext:
+      runAsNonRoot: false
```

Comment:
These values do not match what you have in the comments.

Reply:
I updated the comments.

Reply:
The comment is still the same.